zebra/zapi_msg.c

```diff
@@ -619,6 +619,8 @@ int zsend_redistribute_route(int cmd, struct zserv *client, const struct route_n
 		return -1;
 	}
 
+	assert(stream_get_endp(s) <= stream_size);
```

Comment:
Do you think this can happen? I don't think the stream 's' will allow itself to overrrun...

Reply:
you are right, taken into account